configuration-as-code-examples/cloudbees-assurance-plugin/casc_enabled.yaml

```diff
@@ -0,0 +1,11 @@
+# This file shows how to enable the CloudBees Assurance Program using Configuration as Code.
+# With this configuration, the security warnings for the core and the plugins are enabled.
```

Comment:
WDYT?
```suggestion
# With this configuration, the security warnings from Jenkins and its plugins are enabled.
```

Reply:
I don't think we should mentioned Jenkins when it's a CloudBees product.

Reply:
```suggestion
# With this configuration, the security warnings for the product and the plugins are enabled.
```